src/styles/typography.scss

```diff
@@ -92,6 +92,13 @@
   line-height: 24px;
 }
 
+%face-sans-17-regular {
+  font-family: $font-sans;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 125%;
```

Comment:
why it has % instead of px if we have px in all other fonts?

Reply:
I suggest to use em instead of %. But works similar.
In general I don't think %/em for the line-height as something bad. In many cases it's even better because scales with the font size.